cmd/maya-nodebot/app/command/mcrd.go

```diff
@@ -0,0 +1,40 @@
+package command
+
+import (
+	"errors"
+	"github.com/openebs/maya/cmd/maya-nodebot/mcrd"
+	"github.com/openebs/maya/pkg/util"
+	"github.com/spf13/cobra"
+)
+
+type CmdStartOptions struct {
+	kubeconfig string
+}
+
+//NewSubCmdIscsiLogin logs in to particular portal or all discovered portals
+func NewCmdStart() *cobra.Command {
+	options := CmdStartOptions{}
+	//var target string
+	getCmd := &cobra.Command{
+		Use:   "start",
+		Short: "crd",
```

Comment:
Put a short and long description of `start` command

Reply:
i will change it